package.json

```diff
@@ -5,7 +5,7 @@
   "main": "app.js",
   "scripts": {
     "beautify": "js-beautify --config .js-beautify.json -r *.js {config,models,routes,serverjs,views,public/js}/**/*.js",
-    "start": "node app",
+    "start": "nodemon",
```

Comment:
This will fail to run the app in production because nodemon is a dev dependency, and most production deploys do not include the dev dependencies. (don't know exactly how this app is deployed, so maybe it does!)

IMO, you should leave this as is and create a new script for development:

```json
"dev": "nodemon"
```

Reply:
Also, this seems to me like a separate PR from the `dotenv` change you're making.

Reply:
that's fair... I'll remove this change @crookedneighbor :-)